Orange/widgets/unsupervised/owkmeans.py

```diff
@@ -39,7 +39,7 @@ class OWKMeans(widget.OWWidget):
 
     resizing_enabled = False
 
-    k = Setting(8)
+    k = Setting(3)
```

Comment:
I was following task 321 (Default parameters should be 3 fixed clusters and automatic apply.)